scripts/ci/kurtosis.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+install() {
+    # https://docs.kurtosis.com/install/
+    echo "Installing Kurtosis..."
+    
+    echo "deb [trusted=yes] https://apt.fury.io/kurtosis-tech/ /" | sudo tee /etc/apt/sources.list.d/kurtosis.list
+    sudo apt update
+    sudo apt install -y kurtosis-cli
+    
+    # Validate installation
+    if which kurtosis > /dev/null; then
+        echo "✅ Kurtosis installation completed and verified!"
+    else
+        echo "❌ Kurtosis installation failed. 'kurtosis' command not found in PATH"
+        return 1
+    fi
+}
+
+install_contender() {
+    cargo install --git https://github.com/flashbots/contender --bin contender --force
```

Comment:
If you accept the other suggestion:
```suggestion
    sudo apt-get update
    sudo apt-get install -y libsqlite3-dev libfontconfig1-dev libfontconfig
    cargo install --git https://github.com/flashbots/contender --bin contender --force
```